### ISSUE TITLE
Registration access main activity without login

### ISSUE THREAD

**Issue by janfiedler** [dri94/AllStorj#22]
Hi,
I found small bug.
When you on main login screen click on Register and than click on back button,
you are in main activity of AllStorj. Now i can go to settings, ... when click on Buckets offcorse I get error and crash of application.

I hope I found soon some time to help you contribute with developing if you are interested.
I have same idea about this aplication but to little time do it on my own.

**Comment by adevezin** [comment]
Yes this fix has been merged into code. However i am doing some QA before next release.

I would love some help. Feel free to create some PR's anytime.
